docs/src/main/paradox/features/documentation-overlay.md

```diff
@@ -0,0 +1,23 @@
+Documentation overlay
```

Comment:
The header can remain "Documentation overlay", but I think the file should be renamed to something shorter like `overlay.md`, or just be merged with the multi configuration page.